runner.py

```diff
@@ -0,0 +1,68 @@
+from character import Character
+from wordle import playWordle
+from item import Item0
+from steps import stepcounter
+import steps 
+
+
+flashlight = Item0("Flashlight", "Light")
+
+Map = Item0("Map", "Direction")
```

Comment:
Map needs to be lowercase because it's a protected variable name.
```suggestion
map = Item0("Map", "Direction")
```